### PR TITLE
Tag 40

### PR DESCRIPTION
**Pull request by flapek**
Add to README instruction how to generate sql script to generate database

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,23 +10,37 @@ You need install:
 - [SQL server](https://www.microsoft.com/pl-pl/sql-server/sql-server-downloads)
 - [Chrome](https://www.google.com/chrome)
 - [NPM](https://nodejs.org/)
+- [dotnet-ef](https://docs.microsoft.com/en-us/ef/core/cli/dotnet)
 
 ## Usage
 
 1. Clone the repository
```
Comment (pawelsitarz98) [inline README.md:17]: Add link to the repo

File: README.md [hunk 2]
```diff
@@ -10,23 +10,37 @@ You need install:
...
 
 ### Backend
 
-2. Open terminal and go to folder [`TagsterWebAPI`](./TagsterWebAPI).
-3. Build and publish application with commands below
+2. Open terminal and go to folder [TagsterWebAPI](./TagsterWebAPI).
+3. Generate sql srcipt to initialize database:
+
+- go to [TagsterWebAPI](./TagsterWebAPI/src/TagsterWebAPI) folder with .Net project
+- open [appsettings.json](./TagsterWebAPI/src/TagsterWebAPI/appsettings.json) and update `defaultConnection` for section `connectionStrings`
```
Comment (pawelsitarz98) [inline README.md:25]: -Add information how you can obtain connection string from your database. 
-Add information on what line is section to update

File: README.md [hunk 3]
```diff
@@ -10,23 +10,37 @@ You need install:
...
+
+- use generated script to initialize database
+
+4. Build and publish application with commands below
 
 ```shell
 dotnet build -c Release -o Publish
 ```
 
-4. Go to `Publish` folder.
```
Comment (pawelsitarz98) [inline README.md:27]: Add example of the path

**Review by pawelsitarz98 — COMMENTED** [review]
Update informations using my comments.

File: README.md [hunk 4]
```diff
@@ -35,15 +49,19 @@ TagsterWebAPI.exe
...
+
 ```shell
-npm install 
+npm install
 ```
+
 4. Run local development server using:
+
 ```shell
 npm run serve
```
Comment (kacperantoniak) [inline README.md:61]: shouldn't it be "npm run server"?

**Pull request merged**